editor/bundle/src/main/webapp/app-root/components/page/editor/html.html

```diff
@@ -3,7 +3,8 @@
     data-sly-use.editorConfig="io.wcm.caconfig.editor.model.EditorConfiguration"
     data-sly-use.clientLib="/libs/granite/sightly/templates/clientlib.html"
     data-sly-test="${editorConfig.enabled}"
-    ng-app="io.wcm.caconfig.app">
+    ng-app="io.wcm.caconfig.app"
+    lang="${editorConfig.language}">
```

Comment:
I had originally tried `${currentPage.language.language}`, but this wasn't returning anything (I only tried AEM6.2).

But maybe there is another OOTB way to retrieve the current language, without adding it to EditorConfig?

Reply:
i've changed the code to make sure we use the AEM UI language instead of the content language (which was a bug in caconfig editor all the time anyway).